src/internal/EnhancedButton.js

```diff
@@ -117,9 +101,8 @@ const EnhancedButton = React.createClass({
     listenForTabPresses();
   },
 
-  componentWillReceiveProps(nextProps, nextContext) {
+  componentWillReceiveProps(nextProps) {
     this.setState({
-      muiTheme: nextContext.muiTheme || this.state.muiTheme,
```

Comment:
I think it became an unnecessary setState call when the setting of the muiTheme key line got removed globally. No worries, all fixed now. :smile: